src/pymmcore_widgets/device_properties/_device_property_table.py

```diff
@@ -123,8 +123,16 @@ def _rebuild_table(self) -> None:
         self.clearContents()
         props = list(self._mmc.iterProperties(as_object=True))
         self.setRowCount(len(props))
+
+        # Retrieve the palette
+        palette = self.palette()  # This pulls the current theme colors
+
+        read_only_color = palette.color(QPalette.ColorRole.Dark)
+        pre_init_color = palette.color(QPalette.ColorRole.Dark).darker(130)
+
         for i, prop in enumerate(props):
-            item = QTableWidgetItem(f"{prop.device}-{prop.name}")
+            extra = " (pre-init)" if prop.isPreInit() else ""
```

Comment:
i'm afraid of the manual styling not working.  I think I would pretty much remove all the styling from this PR and just keep this line, but use:

```suggestion
            extra = " 🅿" if prop.isPreInit() else ""
```

Reply:
> i'm afraid of the manual styling not working in some (as of yet unseen) circumstance. 

<img width="897" alt="Screenshot 2024-11-06 at 9 48 50 PM" src="https://github.com/user-attachments/assets/c6711b29-164a-4429-aa98-853149620993">